src/app/core/logging/README.md

```
@@ -0,0 +1,32 @@
+This Guide considers logging of errors and information that is necessary for developers in debugging and monitoring.
+Refer to the _Display Dialogs and Notifications_ guide
+regarding ways to inform the user about errors or other events.
+
+## Logging things
+
+Production code (whatever is merged into the `master` branch) should not contain any calls to `console.log()`.
+Instead, to log information for analysis and debugging (without explicit notification to the user)
+use the `Logging` object.
+This way, all logging is done in a consistent manner and can be transferred to the remote monitoring or saved in some other way:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)